packages/web-components/src/components/masthead/masthead-l1.ts

```diff
@@ -60,19 +60,25 @@ function handleDropdownClose(event: FocusEvent | KeyboardEvent) {
       }
       case 'focusout': {
         const { relatedTarget } = event as FocusEvent;
```

Comment:
is this syntax related to this? https://stackoverflow.com/a/33798750

Reply:
Yep, this is [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment). While it's not any more efficient when you need a single item, it's super handy when you need to pull a bunch of stuff out at once. For example:

```
const {
  relatedTarget,
  currentTarget,
  type
} = event;
```